test/README.md

```diff
@@ -27,19 +27,10 @@ The configuration of the Pull Request trigger is [in the config.json](https://gi
 
 This snippet tells Test Infra to run the [test/e2e.sh](https://github.com/kubernetes/charts/blob/master/test/e2e.sh)
 when testing is triggered on a pull request. The e2e.sh script will use the [Charts test image](https://github.com/kubernetes/charts/blob/master/test/Dockerfile)
-to run the [test/changed.sh](https://github.com/kubernetes/charts/blob/master/test/changed.sh) script. This script
+to run the [chart_test.sh](https://github.com/kubernetes-helm/chart-testing/blob/master/chart_test.sh) script. This script
```

Comment:
this chart_test.sh link is broken

Reply:
The links should work once https://github.com/kubernetes-helm/chart-testing/pull/1 is merged.